lib/install/management/create.go

```diff
@@ -46,15 +46,23 @@ func (d *Dispatcher) CreateVCH(conf *config.VirtualContainerHostConfigSpec, sett
 		return err
 	}
 
-	if err = d.createAppliance(conf, settings); err != nil {
+	d.applianceID, err = d.createAppliance(conf, settings)
```

Comment:
d.applianceID will be affected here in error case as well. Is it a desirable behavior?
I would not assign any value to d.applianceID in case of an error.
It seems like that it doesn't really matter, but is from readability stand point it is a little confusing.